HOMEWORK_15/EX1.CPP

```diff
@@ -0,0 +1,240 @@
+#include <iostream>
+#include <algorithm>
+#include <string>
+#include <vector>
+
+struct Student
+{
+    std::string name;
+    int marks[4];
```

Comment:
replace `4` with const variable and use it instead everywhere

Reply:
Done